src/common/geo/src/wkb.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::io::Cursor;
+
+use scroll::Endian;
+use scroll::IOread;
+use scroll::IOwrite;
+
+const WKB_XDR: u8 = 0;
+const WKB_NDR: u8 = 1;
+const WKB_Z: u32 = 0x80000000;
+const WKB_M: u32 = 0x40000000;
+const WKB_SRID: u32 = 0x20000000;
+const WKB_TYPE_MASK: u32 = 7;
+
+pub fn read_wkb_header(mut raw: &[u8]) -> Result<WkbInfo, String> {
```

Comment:
We already have some related functions to parse `wkb` at this file https://github.com/datafuselabs/databend/blob/main/src/common/io/src/geometry.rs
It is better to use existing functions first, or move those new functions to this file.

Reply:
I known. There's no need to parse the entire wkb just to read srid. A wkb may contain dozens or hundreds of points.

Reply:
please move those functions to `common/io` package, seems not need a new package for some functions.

Reply:
we need a crate like https://github.com/cockroachdb/cockroach/tree/master/pkg/geo

Reply:
For now, it's just a few simple utility functions, we can add a separate package in the future if we have some geo related features.